var/spack/repos/builtin/packages/py-backports-tempfile/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBackportsTempfile(PythonPackage):
+    """This package provides backports of new features in Python’s tempfile
+    module under the backports namespace
+    """
+
+    homepage = "https://github.com/PiDelport/backports.tempfile"
+    url      = "https://pypi.io/packages/source/b/backports.tempfile/backports.tempfile-1.0.tar.gz"
+
+    version('1.0', sha256='1c648c452e8770d759bdc5a5e2431209be70d25484e1be24876cf2168722c762')
+    version('1.0rc1', sha256='1492e2fd8e8abcae6a9c9e78edc52aae4234937c5d462fd3ce0e29c1a4687d3b')
```

Comment:
I wouldn't add `1.0rc1` since it's a release candidate. 1.0 should be more stable.